src/Miru/Queuing/MiruCron.cs

```diff
@@ -0,0 +1,159 @@
+        /// <param name="hour">The hour in which the schedule will be activated (0-23).</param>
+        /// <param name="minute">The minute in which the schedule will be activated (0-59).</param>
+        public static string Monthly(int day, int hour, int minute) =>
+            $"{(object) minute} {(object) hour} {(object) day} * *";
+
+        /// <summary>
+        /// Returns cron expression that fires every year on Jan, 1st at 00:00 UTC.
+        /// </summary>
+        public static string Yearly() => MiruCron.Yearly(1);
+
+        /// <summary>
+        /// Returns cron expression that fires every year in the first day at 00:00 UTC
+        /// of the specified month.
+        /// </summary>
+        /// <param name="month">The month in which the schedule will be activated (1-12).</param>
+        public static string Yearly(int month) => MiruCron.Yearly(month, 1);
+
+        /// <summary>
+        /// Returns cron expression that fires every year at 00:00 UTC of the specified
+        /// month and day of month.
+        /// </summary>
+        /// <param name="month">The month in which the schedule will be activated (1-12).</param>
+        /// <param name="day">The day of month in which the schedule will be activated (1-31).</param>
+        public static string Yearly(int month, int day) => MiruCron.Yearly(month, day, 0);
+        
+        /// <summary>
+        /// Returns cron expression that fires every year at 00:00 UTC of the specified
+        /// month and day of month.
+        /// </summary>
+        public static string YearlySinceToday() => MiruCron.Yearly(DateTime.Now.Month, DateTime.Now.Day, 0);
+
+        /// <summary>
+        /// Returns cron expression that fires every year at the first minute of the
+        /// specified month, day and hour in UTC.
+        /// </summary>
+        /// <param name="month">The month in which the schedule will be activated (1-12).</param>
+        /// <param name="day">The day of month in which the schedule will be activated (1-31).</param>
+        /// <param name="hour">The hour in which the schedule will be activated (0-23).</param>
+        public static string Yearly(int month, int day, int hour) => MiruCron.Yearly(month, day, hour, 0);
+
+        /// <summary>
+        /// Returns cron expression that fires every year at the specified month, day,
+        /// hour and minute in UTC.
+        /// </summary>
+        /// <param name="month">The month in which the schedule will be activated (1-12).</param>
+        /// <param name="day">The day of month in which the schedule will be activated (1-31).</param>
+        /// <param name="hour">The hour in which the schedule will be activated (0-23).</param>
+        /// <param name="minute">The minute in which the schedule will be activated (0-59).</param>
+        public static string Yearly(int month, int day, int hour, int minute) =>
+            $"{(object) minute} {(object) hour} {(object) day} {(object) month} *";
```

Comment:
Maybe it doesn't need to convert to object? I think `$"{minute} {hour} {day} {month} *"` might work. Would you change in other places as well?